src/Microsoft.VisualStudio.ProjectSystem.Managed/ProjectSystem/Properties/InterceptedProjectProperties/DebugPropertyPage/ActiveLaunchProfileNameValueProvider.cs

```diff
@@ -50,9 +50,9 @@ public override Task<string> OnGetUnevaluatedPropertyValueAsync(string propertyN
         /// </remarks>
         private async Task<string> GetPropertyValueAsync()
         {
-            ILaunchSettings launchSettings = await _launchSettings.WaitForFirstSnapshot(Timeout.Infinite);
+            ILaunchSettings? launchSettings = await _launchSettings.WaitForFirstSnapshot(Timeout.Infinite);
 
-            return launchSettings.ActiveProfile?.Name ?? string.Empty;
+            return launchSettings!.ActiveProfile?.Name ?? string.Empty;
```

Comment:
Perhaps add a comment as to why this will not be null. Rather than suppress, consider validating. I really dislike using `!` as it carries no context and looks like design flaw. eg:

```suggestion
            // Infinite timeout means this will not be null
            Assumes.NotNull(launchSettings);
            
            return launchSettings.ActiveProfile?.Name ?? string.Empty;
```

Alternatively, create an overload (or extension method) without a timeout that returns a non-null instance. That way we don't have to replicate this comment in loads of places through the code.